### PR TITLE
Add skywalking log exporter

### PR DESCRIPTION
**Pull request by liqiangz**
Description: Add skywalking log exporter -- First PR

Link to tracking Issue: apache/skywalking#6209

Testing:

logdata_to_logrecord_test.go

Documentation:
exporter/skywalkingexporter/README.md

### CODE REVIEW

File: exporter/skywalkingexporter/README.md
Comment (jpkrohling) [inline exporter/skywalkingexporter/README.md:3]:
```suggestion
Exports data via gRPC using [skywalking-data-collect-protocol](
```

File: exporter/skywalkingexporter/README.md
Comment (jpkrohling) [inline exporter/skywalkingexporter/README.md:13]:
```suggestion
- `endpoint` (no default): host:port to which the exporter is going to send SkyWalking log data,
```

All the mentions to the project name should use the right capitalization.

File: exporter/skywalkingexporter/README.md
Comment (jpkrohling) [inline exporter/skywalkingexporter/README.md:1]:
```suggestion
# SkyWalking gRPC Exporter
```

File: exporter/skywalkingexporter/README.md
Comment (jpkrohling) [inline exporter/skywalkingexporter/README.md:36]:
```suggestion
    insecure: true
```

File: exporter/skywalkingexporter/go.mod
Comment (jpkrohling) [inline exporter/skywalkingexporter/go.mod:3]: Please, double-check this. I think we moved to 1.17 for this repo already.

File: exporter/skywalkingexporter/logdata_to_logrecord.go
Comment (jpkrohling) [inline exporter/skywalkingexporter/logdata_to_logrecord.go:37]: Please, follow the contribution guidelines for new components, specifically the part about breaking the work into multiple PRs.

**Comment by liqiangz** [comment]
Updated, Could you please have a review on this.  @jpkrohling

**Review by jpkrohling — APPROVED** [review]

**Comment by liqiangz** [comment]
Could you please trigger the ci.  @jpkrohling

File: exporter/skywalkingexporter/config.go
Comment (bogdandrutu) [inline exporter/skywalkingexporter/config.go:31]: There is one in the QueueSettings

**Comment by liqiangz** [comment]
@bogdandrutu Updated. Could you please have a review again.

File: exporter/skywalkingexporter/config_test.go
Comment (bogdandrutu) [inline exporter/skywalkingexporter/config_test.go:46]: Please rebase and fix breaking changes related to some function names.

**Review by bogdandrutu — APPROVED** [review]

**Comment by liqiangz** [comment]
@bogdandrutu Updated.

**Comment by tigrannajaryan** [comment]
@liqiangz please add yourself and anyone else who you believe can help maintain this component to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS

**Review by tigrannajaryan — APPROVED** [review]
LGTM, can be merged once codeowners is updated.

**Comment by liqiangz** [comment]
> @liqiangz please add yourself and anyone else who you believe can help maintain this component to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS

@tigrannajaryan  Updated.

**Review by tigrannajaryan — APPROVED** [review]
LGTM.

**Pull request merged**
